shared-libs/couch-request/src/couch-request.js

```diff
@@ -1,10 +1,12 @@
 const environment = require('@medic/environment');
 const path = require('path');
+const os = require('os');
 let asyncLocalStorage;
 let requestIdHeader;
 
 const JSON_HEADER_VALUE = 'application/json';
 const CONTENT_TYPE = 'content-type';
+const CHT_AGENT = 'CHT';
```

Comment:
@dianabarsan - what do you think about this suggestion?

```suggestion
const CHT_AGENT = 'Community Health Toolkit';
```

My thinking is that the if the CHT is misconfiguration and starts accidentally spamming a remote system and the operator there wants to know what `CHT` is it might be hard to look up.  `Community Health Toolkit` will more quickly lead them us.

Don't feel strongly about this!

Reply:
Hey @mrjones-plip  Yes, I think we can have this for more clarity at operator end.

Reply:
@shashi-sah2003, if you agree with @mrjones-plip's suggestion, please commit it and resolve this conversation! Thank you!